core/jazz_t-vault/components/utils/vault.js

```diff
@@ -124,11 +125,12 @@ function deleteSafe(safename, configData, vaultToken, onComplete) {
     logger.debug("deleteSafe response: " + JSON.stringify(response));
     if (response.statusCode && response.statusCode === 200) {
       logger.debug("Successfully deleted safe details: " + JSON.stringify(body));
```

Comment:
We are deleting safe here, not safe details. Make sure message is: ` Successfully deleted safe:`